From 9fe9adc6025efa57a15065de4438c41743a9d0fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A5rten=20Kongstad?= <amhk@google.com>
Date: Fri, 20 Feb 2026 08:45:06 +0000
Subject: [PATCH] Fix compilation error after 37.0 finalization

Fix the following error:

  ERROR: [...]/AccessBluetoothOnCommand.kt:86:20 \
         Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type 'BluetoothLeScanner?'.

Bug: 480974361
Test: m out/soong/.intermediates/packages/modules/Permission/tests/cts/permissionui/UsePermissionApp30WithBluetooth/CtsUsePermissionApp30WithBluetooth/android_common/kotlin/CtsUsePermissionApp30WithBluetooth.jar
Flag: NONE platform SDK finalization
Change-Id: Id4e97e9735c197207ca9ca0c60fb55125e156aa3
---
 .../permissionui/cts/usepermission/AccessBluetoothOnCommand.kt  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/cts/permissionui/UsePermissionApp30WithBluetooth/src/android/permissionui/cts/usepermission/AccessBluetoothOnCommand.kt b/tests/cts/permissionui/UsePermissionApp30WithBluetooth/src/android/permissionui/cts/usepermission/AccessBluetoothOnCommand.kt
index b19f6236230..cae4c56742c 100644
--- a/tests/cts/permissionui/UsePermissionApp30WithBluetooth/src/android/permissionui/cts/usepermission/AccessBluetoothOnCommand.kt
+++ b/tests/cts/permissionui/UsePermissionApp30WithBluetooth/src/android/permissionui/cts/usepermission/AccessBluetoothOnCommand.kt
@@ -83,7 +83,7 @@ class AccessBluetoothOnCommand : ContentProvider() {
                 }
 
             Log.v(LOG_TAG, "call() - startScan...")
-            scanner.startScan(scanCallback)
+            scanner!!.startScan(scanCallback)
 
             // Wait a few seconds to figure out what we actually observed
             Log.v(LOG_TAG, "call() - sleep...")
-- 
2.53.0.345.g96ddfc5eaa-goog

